attachment_preview/static/src/xml/attachment_preview.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t
+        t-name="Attachment"
+        t-inherit="mail.AttachmentCard"
+        t-inherit-mode="extension"
+        owl="1"
+    >
+        <xpath expr="//div[hasclass('o_AttachmentCard_aside')]" position="before">
+            <div
+                class="o_AttachmentCard_aside position-relative overflow-hidden"
+                t-att-class="{ 'o-has-multiple-action d-flex flex-column': !attachmentCard.attachmentList.composerView and attachmentCard.attachment.isEditable }"
+            >
+                <div
+                    t-if="attachmentCard.attachment.downloadUrl"
+                    class="o_AttachmentCard_asideItem d-flex justify-content-center align-items-center ml4 o_attachment_preview"
```

Comment:
o_attachment_preview should be removed here

Reply:
it's removed!